### DISCUSSION TITLE
winget.exe

### DISCUSSION THREAD

**Discussion by Tiger348** [microsoft/winget-cli#2367] [PowerShell]
I'm seeking a workaround for the winget app. When using Powershell on WIN10 ......once Powersell Admin launches and I type winget my system requests  "how do you want to open this file" and prompts a pop up .

If I input winget.exe there's no issue 

When I use the cmd conmand it does not?

Is ther a setting that I've missed

Appreciate any assistance from the community.

ajohn

**Comment by denelon** [comment]
This looks like it could an issue with a corrupted path environment variable:
* https://github.com/microsoft/winget-cli/issues/2391
